src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1137,6 +1137,14 @@ internal override IOperation GetOperationWorker(CSharpSyntaxNode node, Cancellat
                 return null;
             }
 
+            // Nullable suppressions are not directly represented in the bound tree. Rather, they are set
+            // as flags on the bound node underlying the node. Therefore, there is similarly no representation
+            // in the IOperation tree, and we should retrieve the IOperation node underlying the suppression.
+            if (node.IsKind(SyntaxKind.SuppressNullableWarningExpression))
+            {
+                node = ((PostfixUnaryExpressionSyntax)node).Operand;
```

Comment:
@AlekseyTs I'd like your opinion on this change. Currently, we don't attempt to adjust the node being used here at all, and since suppressions don't exist in the bound tree we don't return an IOperation node for the syntax. I made the change in semantic model to adjust the syntax like we do for `GetTypeInfo`, but I'm slightly on the fence about it since it would be the first such change. I feel like this is the appropriate thing to do since GetTypeInfo explicitly adjusts for this, but I could be persuaded against this and just doing this adjustment in my test helper.